### ISSUE TITLE
Outer Wingflex

### ISSUE THREAD

**Issue by jdouglas77** [flybywiresim/aircraft#9247]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-10-31T15:54:41+00:00",
    "ref": "refs/tags/v0.12.0",
    "sha": "2fe89113291f2478347dde0dc3228b55dc924f74",
    "actor": "FoxtrotSierra6829",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.0",
    "version": "a380x-v0.12.0-rel.2fe8911"
}
```

### Describe the bug

The entire wing but predominantly outer section doesn't flex as much as in the air. Ground flex is amazing.

### Expected behavior

Around 10-15% more flex in outer portion of the wing

### Steps to reproduce

Set a flight up, takeoff and climb to FL400. Ensure weight matches that of picture below.

### References (optional)

![IMG_0994](https://github.com/user-attachments/assets/a3e576a3-f2f7-41b9-ae50-633f4e3b5591)
![1250410_20241101160700_1](https://github.com/user-attachments/assets/bde298d7-ab25-4777-9194-4dd21750ace0)
![1250410_20241101160709_1](https://github.com/user-attachments/assets/1f8c0259-94a5-4fbe-8c30-2e28dc1640c4)

**Comment by C-Schaffhausen** [comment]
Agreed, still isn't quite flexing enough imo. In the ground take-off roll the wing kept being bent down as well, irl I think the wing starts raising at 80-100kts

**Comment by A359Flex** [comment]
<img width="1280" alt="Microsoft Flight Simulator Screenshot 2024 10 30 - 14 18 11 79" src="https://github.com/user-attachments/assets/675dea4b-525c-4822-bf36-4034dcaaf061">
![A380-Emirates-Golf-Oscar-Romeo](https://github.com/user-attachments/assets/95861759-6def-4a4c-80c0-9dfe30495bc4)
![shutterstock_112881910-1](https://github.com/user-attachments/assets/75088d08-246b-492c-aa58-4574c8e9fb14)

Noticeable lack of structural flexing on the outboard 2/3rds of the wing when subjected to lift force and resultant wing loading during rotation and in situations of high pitch attitudes.

**Comment by AlonsoJCY** [comment]
![wing_flex](https://github.com/user-attachments/assets/b5bebfe3-2baf-4416-b658-0800bc8e257a)

More evidence!
LHS is simulator; RHS is IRL.
LHS from PVG to PEK; RHS from PVG to FRA
Even if the fuel load on the RHS is greater than on the LHS, RHS still has more wing flex.

**Comment by ksleungac** [comment]
Hi, I would like to add some references on the outer wingflex situation. It is on the ground though. [discord ](https://discord.com/channels/738864299392630914/864211349550268447/1305037252275212398)

**Comment by alepouna** [comment]
Just reposting this from the relevant PR for future commentators: 

> Reminder for observers of this issue, that for flex comparisons with our aircraft and the IRL a380, we need additional data such as the weight, fuel and the "POVs" to match in both comparisons, otherwise it isn't really helpful!

**Comment by B779Flex** [comment]
<img width="1280" height="747" alt="Image" src="https://github.com/user-attachments/assets/6d4d5ab5-79ef-453b-af52-eb5fda9af3d2" />

Latest stable update shows the issue hasn't been fixed yet. Both Midboard and Outboard flex is too minuscule to be discernible.

![Image](https://github.com/user-attachments/assets/903b0f7b-238e-46f6-864c-14be70c6b569)

**Comment by C-Schaffhausen** [comment]
I'd like to add #9612 to this post, the wing droop is too extreme on the ground as well.

**Comment by C-Schaffhausen** [comment]
Hello, may I ask if there's any update on this issue? @crocket63 Any chance to review the wingflex model? Much appreciated.
